scripts/zones/Halvung/mobs/Copper_Borer.lua

```diff
@@ -4,6 +4,20 @@
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+    mob:addStatusEffect(xi.effect.BLAZE_SPIKES, 10, 0, 0)
+end
+
+entity.onMobSpawn = function(mob)
+    -- Mob needs to be in the correct state in order to use thermal pulse
+    mob:setAnimationSub(0)
```

Comment:
shouldn't it just be 0 in pool? and then every spawn should start as that anyway.